src/config.h

```diff
@@ -16,6 +16,7 @@
 	#define PROJECT_NAME_C "Minetest"
 	#define STATIC_SHAREDIR ""
 	#define VERSION_STRING STR(VERSION_MAJOR) "." STR(VERSION_MINOR) "." STR(VERSION_PATCH) STR(VERSION_EXTRA)
+	#define ENABLE_GLES 1
```

Comment:
Shouldn't it get into PR?

Reply:
Yeah this looks like it shouldn't be here.